### ISSUE TITLE
Uncaught TypeError: Invalid Point: (5, NaN)

### ISSUE THREAD

**Issue by Harvey-Lu** [atom-community/linter#1008]
[Enter steps to reproduce below:]

1. The error occurs immediately when I open a .tex file, with package linter-chktex.

**Atom Version**: 1.2.3
**System**: Windows 8.1
**Thrown From**: [linter](https://github.com/atom-community/linter) package, v1.11.2

### Stack Trace

Uncaught TypeError: Invalid Point: (5, NaN)

```
At C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\text-buffer\lib\point.js:65

TypeError: Invalid Point: (5, NaN)
    at Function.module.exports.Point.assertValid (C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\text-buffer\lib\point.js:65:15)
    at TextBuffer.module.exports.TextBuffer.clipPosition (C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:980:13)
    at TextBuffer.module.exports.TextBuffer.clipRange (C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:968:20)
    at TextBuffer.module.exports.TextBuffer.markRange (C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\text-buffer\lib\text-buffer.js:693:46)
    at DisplayBuffer.module.exports.DisplayBuffer.markBufferRange (C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\src\display-buffer.js:1065:41)
    at TextEditor.module.exports.TextEditor.markBufferRange (C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\src\text-editor.js:1267:59)
    at C:/Users/lenovo/.atom/packages/linter/lib/editor-linter.js:50:34
    at Emitter.module.exports.Emitter.emit (C:\Users\lenovo\AppData\Local\atom\app-1.2.3\resources\app.asar\node_modules\event-kit\lib\emitter.js:86:11)
    at EditorLinter.addMessage (C:/Users/lenovo/.atom/packages/linter/lib/editor-linter.js:137:20)
    at C:/Users/lenovo/.atom/packages/linter/lib/linter-views.js:104:22
```

### Commands

```
```

### Config

```json
{
  "core": {
    "autoHideMenuBar": true,
    "openEmptyEditorOnStart": false,
    "projectHome": "D:\\Programming\\Github",
    "themes": [
      "atom-material-ui",
      "monokai"
    ],
    "disabledPackages": [
      "build",
      "build-make",
      "vim-mode"
    ]
  },
  "linter": {}
}
```

### Installed Packages

```coffee
# User
atom-material-ui, v0.8.0
color-picker, v2.0.13
external-open, v0.6.1
file-icons, v1.6.12
language-haskell, v1.5.0
language-latex, v0.6.1
language-prolog, v0.10.0
language-scheme, v1.4.0
language-smalltalk, v1.0.2
latex, v0.28.2
latexer, v0.2.7
linter, v1.11.2
linter-chktex, v1.2.3
linter-gcc, v0.4.1
minimap, v4.16.1
monokai, v0.18.0

# Dev
No dev packages
```

**Comment by Harvey-Lu** [comment]
I found that previously there were 4 more warnings from `linter-chktex` that I used to ignore.
And when I opened the `source.tex` file, a new file called roughly `source.tex 54:17` was created. After checking the location, I found that the codes that generates the 4 warnings were exactly there. The codes are:

```tex
%\tablefirsthead{\hline}
%\tablehead{\hline}
%\tablelasttail{\hline}
%\tabletail{\hline}
```

So what would trigger the linter to open a new file perticular to a location where there was some warning about the table lines?
